src/alt/store/AltStore.js

```diff
@@ -55,7 +55,7 @@ class AltStore {
           return actionHandler.call(model, payload.data, payload.action)
         }, payload)
 
-        if (result !== false && !this.preventDefault) this.emitChange()
+        if (result !== false && !this.preventDefault && this.StoreModel.config.shouldEmitChange) this.emitChange()
```

Comment:
@endragor I believe `emitChange` will still be called when `setState` is used [here](https://github.com/goatslacker/alt/blob/master/src/alt/store/index.js#L24). @goatslacker correct me if I am wrong.